pkg/plugins/policies/meshhttproute/xds/configurer.go

```diff
@@ -32,17 +31,14 @@ func (c RoutesConfigurer) Configure(virtualHost *envoy_route.VirtualHost) error
 	}
 
 	for _, match := range matches {
-		rb := &route.RouteBuilder{}
-
-		rb.Configure(route.RouteMustConfigureFunc(func(envoyRoute *envoy_route.Route) {
-			// todo: create configurers for Match and Action
-			envoyRoute.Match = match.routeMatch
-			envoyRoute.Action = &envoy_route.Route_Route{
-				Route: c.routeAction(c.Split),
-			}
-			envoyRoute.TypedPerFilterConfig = map[string]*anypb.Any{}
-			envoyRoute.Name = h
```

Comment:
Those are moved into the RouteBuilder, the name as an argument in the constructor function, the TypedPerFilterConfig is instantiated as an empty map in the Build method before any configurer is applied.